### PR TITLE
Fixing the library title for Sharepoint User Profiles (b/21765085)

### PR DESCRIPTION
**Pull request by mpaggi**
Implementation title for Google Adaptor Sharepoint User Profiles
is the same as for the SharePoint Google Adaptor for SharePoint
Fixing it by setting up two different variables in the build file.

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -259,7 +263,7 @@ lib/plexi submodule or add the the command line argument
         <attribute name="Class-Path" value="${tmp.adaptor.classpath}"/>
         <section name="com/google/enterprise/adaptor/sharepoint/">
           <attribute name="Implementation-Title"
-            value="Google Adaptor for SharePoint"/>
+            value="${adaptor.title}"/>
```
Comment (wiarlawd) [inline build.xml]: It looks like all three of these edited & added attribute elements will now fit on one line.

File: build.xml [hunk 2]
```diff
@@ -23,6 +23,10 @@
     value="com.google.enterprise.adaptor.sharepoint.SharePointAdaptor"/>
   <property name="adaptor2.class"
     value="com.google.enterprise.adaptor.sharepoint.SharePointUserProfileAdaptor"/>
+  <property name="adaptor.title"
+    value="Google Adaptor for SharePoint"/>
```
Comment (wiarlawd) [inline build.xml]: While you're doing this, you could also update these titles to be correct:

Google Search Appliance Connector for SharePoint
Google Search Appliance Connector for SharePoint User Profiles

**Review by wiarlawd — CHANGES_REQUESTED** [review]
A nit and a suggestion.

John L

**Comment by mpaggi** [comment]
Thanks, done.

**Review by wiarlawd — APPROVED** [review]

**Pull request merged**